server/index.js

```diff
@@ -41,6 +39,30 @@ app.get('/api/products', (req, res, next) => {
     });
 });
 
+app.get('/api/products/:productId', (req, res, next) => {
+  const iD = req.params.productId;
+  const sql =
+    `select
+    "productId",
+    "name",
+    "price",
+    "image",
+    "longDescription",
+    "shortDescription"
+    from "products"
+    where "productId" = ${iD}`;
```

Comment:
This is a security vulnerability. You need to use a placeholder like you did for sgt back end `$1`